### PR TITLE
Build new /kubernetes/what-is-kubernetes page

### LINKED ISSUE #7552: [new page] /what-is-k8s - build page

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Build new /kubernetes/what-is-kubernetes page
- Added it to the navigation
- **NOTE** - needs update of top diagram

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/what-is-kubernetes page
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [design](#7551) and the [copy doc](https://docs.google.com/document/d/19ZAsyujRHjOBIKP15F-4Z9L24KpfoYrx0YpL-0gO6sE/edit#)

## Issue / Card

Fixes #7552

### CODE REVIEW

**Comment by meralonso** [comment]
Looking good! I have some comments:

- 'What are containers?' section
New image including titles [here](https://assets.ubuntu.com/v1/3d57842f-diagram-machine-virtualization-vs-containers.svg)
Also, give a bit more space with sentence below, please.

- KEY CONTRIBUTORS TO KUBERNETES 
It looks a bit unbalanced. Canonical logo is too big and the vertical version of Huawei logo is a bit small comparing to others. Could you do something similar to my design and use [this Huawei logo](https://assets.ubuntu.com/v1/80fe0ebe-Huawei-logo.svg), please?
Screenshot 2020-06-05 at 09 50 52

- 'What Kubernetes is not' section
Can you put 3 bullet point on the left column and 2 on the right one, please?

- 'Using Kubernetes for development' section
Titles in the tiles should be h4, I'm ok with or without the line underneath.

- Last section
Bit more space between CTA and orange line, please.

Thanks!

**Comment by pmahnke** [comment]
@meralonso I think I have addressed all your comments.  The logos are a bit off, but a lot of the min and max sizing is in vanilla... worth looking at as a bigger project.

Let me know what you think.

**Comment by meralonso** [comment]
Nice, happy with it! Thanks @pmahnke

File: templates/kubernetes/what-is-kubernetes.html [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+          </li>
+          <li class="p-list__item is-ticked">Mounting and adding storage to run stateful apps
+          </li>
+          <li class="p-list__item is-ticked">Scaling containerised applications and their resources on the fly
+          </li>
+          <li class="p-list__item is-ticked">Declaratively managing services, which guarantees that applications are always running as intended
+          </li>
+          <li class="p-list__item is-ticked">Health-checking and self-healing applications with auto-placement, autorestart, auto-replication and autoscaling
+          </li>
+          <li class="p-list__item is-ticked">Being open source (<a href="https://github.com/kubernetes/kubernetes">all Kubernetes code is on GitHub</a>) and maintained by a large, active community</li>
```
Comment (sowasred2012) [inline templates/kubernetes/what-is-kubernetes.html:427]:
```suggestion
          <li class="p-list__item is-ticked">Being open source (<a class="p-link--external" href="https://github.com/kubernetes/kubernetes">all Kubernetes code is on GitHub</a>) and maintained by a large, active community</li>
```

**Pull request merged**